doc/releases/nautilus.rst

```diff
@@ -3,15 +3,18 @@ v14.2.13 Nautilus
 
 This is the 13th backport release in the Nautilus series. This release fixes a
 regression introduced in v14.2.12, that referred to ceph-mon hosts using dns
-names instead of ip addresses in the `mon_host` param in `ceph.conf`. We
-recommend users to update to this release.
+names instead of ip addresses in the `mon_host` param in `ceph.conf` other than
+a few ceph-volume & RGW fixes. We recommend users to update to this release.
```

Comment:
In addition to bringing some ceph-volume and RGW fixes, this release fixes a serious regression in 14.2.12 which broke configurations where the `mon_host` parameter of `ceph.conf` referred to hosts by their DNS names instead of IP addresses.

Reply:
We recommend that all Nautilus users update to this release.

Reply:
I actually updated and moved this to Notable Changes... sorry forgot ot push... updated now